### PR TITLE
decimal fix

### PR DESCRIPTION
**Pull request by WHYTEWYLL**

### CODE REVIEW

File: models/silver/curated/defi/bridge/silver__bridge_allbridge.sql [hunk 1]
```diff
@@ -48,7 +48,7 @@ outbound_near AS (
         block_timestamp,
         tx_hash,
         args :create_lock_args :token_id :: STRING AS token_address,
-        args :create_lock_args :amount :: INT AS amount_raw,
+        args :create_lock_args :amount   * pow(10, 15 ) :: NUMBER AS amount_raw,
```
Comment (eric-laurello) [inline models/silver/curated/defi/bridge/silver__bridge_allbridge.sql:51]: this isn't a raw amount if it's adjusted. Do all tokens have the same amount of decimals?

Comment (WHYTEWYLL) [reply models/silver/curated/defi/bridge/silver__bridge_allbridge.sql:51]: Yes!

Comment (WHYTEWYLL) [reply models/silver/curated/defi/bridge/silver__bridge_allbridge.sql:51]: Its something specific to allbrige. And how they format the token amounts at the event we use.

**Comment by WHYTEWYLL** [comment]
Added a price mapping where I included Ethereum prices to cover some of the biggest tokens with the most activity.

File: models/silver/curated/defi/bridge/silver__bridge_allbridge.sql [hunk 2]
```diff
@@ -48,7 +48,8 @@ outbound_near AS (
         block_timestamp,
         tx_hash,
         args :create_lock_args :token_id :: STRING AS token_address,
-        args :create_lock_args :amount :: INT AS amount_raw,
+        args :unlock_args :amount :: INT AS amount_raw,
```
Comment (eric-laurello) [inline models/silver/curated/defi/bridge/silver__bridge_allbridge.sql:51]: did you mean to change the source of the amount?

File: models/gold/defi/defi__ez_bridge_activity.sql [hunk 3]
```diff
@@ -38,19 +39,36 @@ labels AS (
...
     GROUP BY
         1,
         2
 ),
+prices_mapping AS (
+    SELECT
+        block_timestamp,
+        CASE
+            WHEN contract_address = '0xf7413489c474ca4399eee604716c72879eea3615' THEN 'apys.token.a11bd.near'
+            WHEN contract_address = '0x3294395e62f4eb6af3f1fcf89f5602d90fb3ef69' THEN 'celo.token.a11bd.near'
```
Comment (eric-laurello) [inline models/gold/defi/defi__ez_bridge_activity.sql:61]: this works for now, but would make more sense to map these into the silver__complete_token_prices logic

File: models/gold/defi/defi__ez_bridge_activity.sql [hunk 4]
```diff
@@ -59,7 +77,7 @@ FINAL AS (
         b.token_address,
         b.amount_unadj,
         l1.symbol,
-        b.amount_unadj / pow(
+        b.amount_adj / pow(
```
Comment (eric-laurello) [inline models/gold/defi/defi__ez_bridge_activity.sql:80]: Just confirming for allbridge we need to adjust in silver and adjust again in gold?

Comment (WHYTEWYLL) [reply models/gold/defi/defi__ez_bridge_activity.sql:80]: yes sir!

File: models/gold/defi/defi__ez_bridge_activity.sql (hunk 3 above)
Comment (WHYTEWYLL) [reply models/gold/defi/defi__ez_bridge_activity.sql:61]: roger that! Agree, in the future makes more sense to move it there.

**Pull request merged**